content/kotlin/concepts/conditionals/conditionals.md

```diff
@@ -0,0 +1,158 @@
+---
+Title: 'Conditionals'
+Description: 'Conditionals take an expression, which is code that evaluates to determine a value, and does something if it is true or false.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Conditionals'
+  - 'Control Flow'
+  - 'Switch'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+**Conditionals** are commands in Kotlin that handle decisions. With conditionals, code is dynamic, which means that it can behave differently given a different condition. In conditionals statements different code blocks can be written which can execute when the condition is met.
```

Comment:
```suggestion
**Conditionals** are commands in Kotlin that handle decisions. With conditionals, code is dynamic, which means that it can behave differently given a different condition. In conditional statements different code blocks can be written which can execute when the condition is met.
```